editor/import/resource_importer_texture.cpp

```diff
@@ -227,21 +227,8 @@ void ResourceImporterTexture::_save_stex(const Ref<Image> &p_image, const String
 	f->store_8('S');
 	f->store_8('T'); //godot streamable texture
 
-	bool resize_to_po2 = false;
-
-	if (p_compress_mode == COMPRESS_VIDEO_RAM && p_force_po2_for_compressed && (p_mipmaps || p_texture_flags & Texture::FLAG_REPEAT)) {
-		resize_to_po2 = true;
-		f->store_16(next_power_of_2(p_image->get_width()));
-		f->store_16(p_image->get_width());
-		f->store_16(next_power_of_2(p_image->get_height()));
-		f->store_16(p_image->get_height());
-	} else {
-		f->store_16(p_image->get_width());
-		f->store_16(0);
-		f->store_16(p_image->get_height());
-		f->store_16(0);
-	}
-	f->store_32(p_texture_flags);
```

Comment:
These are the only changes I'm not confident with as they impact all formats, so I want @reduz to review this more closely.

The rest of the changes seem good though (with some obvious bugfixes like in `drivers/gles2` or `platform/iphone/export/export.cpp`), so I'd suggest splitting the texture importer change to a dedicated PR and remove it here, so that the rest can be merged first.